src/components/common/animation-control/animation-slider.js

```diff
@@ -127,9 +114,17 @@ const AnimationControlFactory = () => {
     };
 
     _nextFrame = () => {
-      const {domain} = this.props.animation;
-      this._currentStep > domain[1] - domain[0] - 1 ? 0 : (this._currentStep += 3);
-      this.onSlider1Change(this._currentStep);
+      const {currentTime, domain, speed} = this.props.animation;
```

Comment:
`this.animation.speed` is a multiplier. Copy the logic from `time-range-slider`. This should be:
```
    const speed = ((domain[1] - domain[0]) / BASE_SPEED) * this.props.speed;
```

Also, the logic here is hard to follow, simplify it to:

```
// if nextTime exceed domain[1], set to back to domain[0]
const nextTime = currentTime + speed > domain[1] ? domain[0] : currentTime + speed;
this.props.playAnimation(nextTime);
```
That's it, you don't need a _pauseAnimation case